31_service_kibana.yml

```diff
@@ -6,9 +6,8 @@ metadata:
   labels:
     app: kibana
 spec:
-  clusterIP: None # <--
   ports:
-  - port: 5601
+  - port: 80
```

Comment:
Why making this change ?

Reply:
In general, services serve as a good abstraction to simplify the pod internal ports. 
By following this convention (port 80 for http services) we avoid using serviceName + port to interact with services (due port 80 is implicit), so other services or applications can just do http://kibana to interact with this service. 

This is just a convention for http services, this can be reverted and also handled at ingress level, but I tend to think that this is a useful approach to deal with http services.